lib/lambda/setupIndex.test.ts

```diff
@@ -20,63 +20,39 @@ describe("handler", () => {
   });
 
   it("should create and update indices without errors", async () => {
-    await handler(mockEvent, null, mockCallback);
-
-    expect(os.createIndex).toHaveBeenCalledTimes(7);
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-main",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-changelog",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-types",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-subtypes",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-cpocs",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-insights",
-    );
-    expect(os.createIndex).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-legacyinsights",
-    );
-
-    expect(os.updateFieldMapping).toHaveBeenCalledTimes(1);
-    expect(os.updateFieldMapping).toHaveBeenCalledWith(
-      "test-domain",
-      "test-namespace-main",
-      {
-        approvedEffectiveDate: { type: "date" },
-        changedDate: { type: "date" },
-        finalDispositionDate: { type: "date" },
-        proposedDate: { type: "date" },
-        statusDate: { type: "date" },
-        submissionDate: { type: "date" },
-      },
-    );
+    await handler(mockEvent, {} as Context, mockCallback);
```

Comment:
```suggestion
    await handler(mockEvent, expect.anything(), mockCallback);
```

Look, ma! No casting needed!